pkg/executor/executor.go

```diff
@@ -162,12 +161,12 @@ func (e *Executor) runJob(ctx context.Context) bool {
 		return true
 	}
 
-	e.logger.Infof("job picked up by a worker, going to run job: %s", j.Type())
+	e.logger.Infof("job picked up by a worker, going to run job: %s", j.ID())
```

Comment:
By using hash id in logs, since humans have no idea which job has which hash, will it make it harder to  identify which job is running? Maybe show both job type and id?